### PR TITLE
app-arch/libarchive: Added verify-sig integration

### PR DESCRIPTION
**Pull request by jpds**

### CODE REVIEW

File: sec-keys/openpgp-keys-libarchive/openpgp-keys-libarchive-20220324.ebuild [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="OpenPGP keys used by the libarchive project"
+HOMEPAGE="https://www.libarchive.org/"
```
Comment (mgorny) [inline sec-keys/openpgp-keys-libarchive/openpgp-keys-libarchive-20220324.ebuild:7]: Does the homepage list the key used anywhere?

Comment (jpds) [reply sec-keys/openpgp-keys-libarchive/openpgp-keys-libarchive-20220324.ebuild:7]: Unfortunately not, I pulled it out of the `.asc`, and the release commits have the same key ID.

Comment (mgorny) [reply sec-keys/openpgp-keys-libarchive/openpgp-keys-libarchive-20220324.ebuild:7]: Guessed so.

**Review by mgorny — APPROVED** [review]
Thanks. I'll do some reformatting and merge.

**Pull request closed without merging**